python/src/IteratedQuadrature_doc.i.in

```diff
@@ -37,59 +43,16 @@ Create an iterated quadrature algorithm:
 >>> l = [ot.SymbolicFunction(['x'], [' 2+cos(x)'])]
 >>> u = [ot.SymbolicFunction(['x'], ['-2-cos(x)'])]
 
-Draw the graph of the integrand and the bounds:
-
->>> g = ot.Graph('Integration nodes', 'x', 'y', True, 'upper right')
->>> g.add(f.draw([a,a],[b,b]))
->>> curve = l[0].draw(a, b).getDrawable(0)
->>> curve.setLineWidth(2)
->>> curve.setColor('red')
->>> g.add(curve)
->>> curve = u[0].draw(a, b).getDrawable(0)
->>> curve.setLineWidth(2)
->>> curve.setColor('red')
->>> g.add(curve)
-
 Evaluate the integral with high precision:
 
 >>> Iref = ot.IteratedQuadrature(ot.GaussKronrod(100000, 1e-13, ot.GaussKronrodRule(ot.GaussKronrodRule.G11K23))).integrate(f, a, b, l, u)
 
-Evaluate the integral with the default GaussKronrod algorithm, and get
-evaluation points:
-
->>> f = ot.MemoizeFunction(f)
->>> I1 = ot.IteratedQuadrature(ot.GaussKronrod()).integrate(f, a, b, l, u)
->>> sample1 = f.getInputHistory()
->>> print(I1)
-[-25.132...]
->>> n_evals = sample1.getSize()
->>> print(n_evals)
-2116
->>> err = abs(100.0*(1.0-I1[0]/Iref[0]))
->>> print(err)
-2.2...e-14
->>> cloud = ot.Cloud(sample1)
->>> cloud.setPointStyle('fcircle')
->>> cloud.setColor('green')
->>> g.add(cloud)
->>> f.clearHistory()
-
 Evaluate the integral with the default IteratedQuadrature algorithm:
 
->>> I2 = ot.IteratedQuadrature().integrate(f, a, b, l, u)
->>> sample2 = f.getInputHistory()
->>> print(I2)
-[-25.132...]
->>> n_evals = sample2.getSize()
->>> print(n_evals)
-5236
->>> err = abs(100.0*(1.0-I2[0]/Iref[0]))
->>> print(err)
-4.6...e-10
->>> cloud = ot.Cloud(sample2)
->>> cloud.setPointStyle('fcircle')
->>> cloud.setColor('gold')
->>> g.add(cloud)"
+>>> Idefault = ot.IteratedQuadrature().integrate(f, a, b, l, u)
+>>> relative_error = abs(1.0-Idefault[0]/Iref[0])
+>>> print(relative_error)
+4.6...e-10"
```

Comment:
better drop the print, lets not turn this into an unit test

Reply:
I don't understand the compilation problems...

Reply:
doctest fails here because the exponent is wrong:
```
Expected:
    4.6...e-10
Got:
    4.679145959585185e-12
```

Reply:
ok